pkg/server/api/svid/v1/service.go

```diff
@@ -300,6 +312,11 @@ func (s *Service) mintJWTSVID(ctx context.Context, protoID *types.SPIFFEID, audi
 		return nil, api.MakeErr(log, codes.Internal, "failed to get JWT-SVID expiry", err)
 	}
 
+	log.WithFields(logrus.Fields{
+		telemetry.Audience:   audience,
+		telemetry.Expiration: expiresAt.Format(time.RFC3339),
+	}).Debug("Server CA successfully signed JWT SVID")
```

Comment:
Wonder if we shouldn't take this opportunity to change this log line to be consistent with the others?
```suggestion
	}).Debug("Signed JWT SVID")
```

Reply:
I guess I feel inclined to keep the message the same as it is today since the proposed change in wording is semantically equivalent. I would prefer to err on the side of preserving the existing message text that users are already familiar with to prevent potentially breaking any log-based alerting.

Reply:
Fair 'nough!